Actions/RunPipeline/RunPipeline.ps1

```diff
@@ -113,6 +113,38 @@ try {
     $settings = AnalyzeRepo -settings $settings -baseFolder $baseFolder -project $project @analyzeRepoParams
     $settings = CheckAppDependencyProbingPaths -settings $settings -token $token -baseFolder $baseFolder -project $project
 
+    if ($bcContainerHelperConfig.ContainsKey('TrustedNuGetFeeds')) {
```

Comment:
Could this be after the "Repository is empty, exiting" check?

Reply:
Yes